### PR TITLE
878 create tests for weblist 0 1

### PR DESCRIPTION
**Pull request by vladb00mer**
#1624 Delete all Merged branches older than 1 month

### CODE REVIEW

File: jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import static org.testng.Assert.assertEquals;
+
+public class WebListTests {
+    private String element0Text = "0 element";
+    private String element1Text = "1 element";
+    private By locator = new By.ByCssSelector("");
+    private List<WebElement> list;
+    private WebList webList;
+    private WebElement element0;
+    private WebElement element1;
```
Comment (codacy-review) [inline jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java]: Codacy Issue found: [Perhaps 'element1' could be replaced by a local variable.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4816272)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4816272&bid=16016661)

File: jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+public class WebListTests {
+    private String element0Text = "0 element";
+    private String element1Text = "1 element";
+    private By locator = new By.ByCssSelector("");
+    private List<WebElement> list;
+    private WebList webList;
+    private WebElement element0;
```
Comment (codacy-review) [inline jdi-light/src/test/java/io/github/epam/weblisttests/WebListTests.java]: Codacy Issue found: [Perhaps 'element0' could be replaced by a local variable.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4816272)

**Comment by RomanIovlev** [comment]
already implemented

**Pull request closed without merging**